modules/mobile/app/controllers/mobile/v0/payment_history_controller.rb

```diff
@@ -57,6 +58,7 @@ def filter(payments, available_years, validated_params)
           most_recent_year = available_years.first
 
           unless most_recent_year.is_a? Numeric
+            # don't see any instances of this. i think we can remove it. the this were to happen, we'd get a fairly clear error message
             Rails.logger.error('Mobile Payment Error Non Numeric Year', { year_in_error: most_recent_year,
                                                                           available_years: })
```

Comment:
Category: data discovery
Action: remove. it's not happening and would result in a clear error message